### ISSUE TITLE
Bug: uYou "Disable Auto Caption" breaks CC subtitles completely

### ISSUE THREAD

**Issue by hi-im-karma** [arichornlover/uYouEnhanced#451]
Enabling this option prevents subtitles automatically showing all the time, but at the same time prevents the CC button and functionality from working.
![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/a66346fb-6f06-43cd-9818-3d966a4d440f)

https://github.com/arichornlover/uYouEnhanced/assets/73917917/63f9d9f5-b31b-49f8-ba5e-3be39fb33ef1

**Comment by hi-im-karma** [comment]
No I mean it should disable the "Auto" enabling of captions. That's what annoys most people. Especially the Vanced/Revanced community hates it when CC are automatically enabled by YouTube.

It should disable auto-cc and let the user enable/disable it when he wants by tapping the CC button

**Comment by hi-im-karma** [comment]
Or maybe add your own tweak? Just disable automatically showing captions. This super annoying YouTube behavior. Revanced mods all have this as a feature.

**Comment by bolabola6789** [comment]
Workaround if you still want to disable auto captions (I have this problem for long, since the old days before the existence of this repo)
Go to player settings (the cog next to the CC button), select Captions, then turn it off
then redo the steps above, but select the CC language you desire this time instead of turning it off

CC should work now.

**Comment by KarmaBeast** [comment]
I don't get it what are you trying to explain?

I want CC to work just not automatically enabled shown to me. The annoying YT behavior where it automatically shows subtitles on every video.

**Comment by bolabola6789** [comment]
> I don't get it what are you trying to explain?
> 
> I want CC to work just not automatically enabled shown to me. The annoying YT behavior where it automatically shows subtitles on every video.

https://youtube.com/shorts/jej6CX0Og70?feature=share
I screen recorded the steps I do to turn on CC. This is pretty much what I told you to do

**Comment by bolabola6789** [comment]
It also happens on Revanced (uYou but for Android) every now and then, so I
don't think it will be fixed anytime soon. It's not that complicated also.
And this is uYou's feature, not uYou enhanced's, so I don't think it's the
right place to complain.

**Comment by KarmaBeast** [comment]
Who said anything about complaining? I was merely requesting for implementation of seperate uyouenhanced tweak that would only disable the automatic showing of subtitles, but not disable subtitles itself, like the uYou tweak does...

**Comment by KarmaBeast** [comment]
Also you're wrong on Revanced it works as intended:

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/71245ee7-287e-4847-969e-4668e5e13ccb)

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/fc848a99-063c-4c7e-b9f8-b657d4c9af19)

It only disable auto captions. Auto-captions is when subtitles are automatically being forced on and appear on the video. This behavior annoys YouTube users though, as we only want to manually enable subs ourself when we want it.
The Revanced mod doesn't fully disable/break the subtitles functionality. It only disables described behavior that is a huge annoyance to users.

https://github.com/arichornlover/uYouEnhanced/assets/73917917/72210e28-30a1-4af7-8f06-99a65af306c4

**Comment by theoccurrence** [comment]
Have you made an issue about this on uYou's Guthub page already? Because I also just noticed this and it's really annoying. Especially because it was me a long time ago which requested the disable automatic subtitles feature from Miro in the first place 😅

**Comment by KarmaBeast** [comment]
No I don't think I have.

The feature is just broken. It disables the subtitles functionality completely, instead of merely only preventing auto-subtitles from showing when starting a video.

The latter is what the Revanced feature does. Only prevent automatically enabling of subtitles. Its the stock YouTube behavior and massively annoying.

I'm not sure if anyone cares to fix it, and implement a uyouenhanced tweak instead.

**Comment by theoccurrence** [comment]
Yes, it really is annoying. Luckily, I usually don't need subtitles, so this issue doesn't really impact me personally that much, but I get it.

YouTube also started defaulting to the weirdest audio languages, apparently that's a thing now. I noticed this on a Marc Rober video, I think it always wanted to default to arab. Another video (I can't remember who) defaulted to hindi. Both cases were dubbed and not the original audio.

I might have to make another feature request for that, but I'm not sure if it still happens.
